src/components/ErrorBoundary.tsx

```diff
@@ -1,30 +1,48 @@
 import * as Sentry from '@sentry/react';
 import React from 'react';
-import { Error } from './Error';
+import { Error as ErrorComponent, ErrorPropTypes } from './Error';
 import type { ErrorBoundaryProps } from '@sentry/react/types/errorboundary';
 import { ErrorContext } from '../contexts';
 import { SentryError } from '../types';
 
+const Error = ({ children, ...props }: React.PropsWithChildren<ErrorPropTypes>) =>
+  <ErrorComponent data-testid='error-fallback' {...props}>{children}</ErrorComponent>;
+
+export const defaultErrorFallbacks = {
+  'generic': <Error data-testid='error-fallback' />,
+  'SessionExpiredError': <Error heading='Your session has expired'>
+    Please refresh your browser and try again. If this doesn't solve the problem, visit our <a href="https://openstax.secure.force.com/help" target="_blank">Support Center</a>.
+    </Error>
+};
+
+export const getTypeFromError = (error: Error | PromiseRejectionEvent['reason']) => {
+  const { TYPE, name } = error.constructor;
+  return TYPE && typeof TYPE === 'string' ? TYPE : name;
+};
+
 export const ErrorBoundary = ({
   children,
   renderFallback,
-  fallback,
+  fallback = defaultErrorFallbacks['generic'],
   catchUnhandledRejections = true,
   windowImpl = window,
   sentryDsn,
   sentryInit,
+  errorFallbacks = defaultErrorFallbacks,
```

Comment:
instead of setting the default here, you should probably set this to an empty object, then in the function body do 
```
const errorFallbacks = {...defaultErrorFallbacks, ...props.errorFallbacks}
```
so that inputting additional errorFallbacks doesn't clear out the default ones, but the default ones can still be overridden